### ISSUE TITLE
A more newbie friendly example on the readme.md page

### ISSUE THREAD

**Issue by mcapodici** [elastic/elasticsearch-js#893]
## 🚀 Feature Proposal

A more newbie friendly example on the readme.md page

## Motivation

As a new user, I found it hard to get the example working. When searching for a library to use the code not working first time is annoying and might make me try another library or just not bother and use HTTP.

The reason it didn't work is because it tries to do a POST request in a search query, returning a 400 response code. I suggest changing this to a GET request, and putting an example document in the index to begin with.

## Example

Here is an example:

```
const { Client } = require('@elastic/elasticsearch')
const client = new Client({ node: 'http://localhost:9200' })

// promise API
async function main() {

    await client.index({
        index: 'example-index',
        body: { counter: 1 }
    })

    const result = await client.search({
        index: 'example-index',
        q: 'counter:1'
    });

    console.log(result.body.hits.hits[0]);

};

main();
```

**Comment by delvedor** [comment]
Hello! There are a lot of examples in [`docs/examples`](https://github.com/elastic/elasticsearch-js/tree/master/docs/examples), but I agree that these examples can be better surfaced in the README. I'll work on this.

> The reason it didn't work is because it tries to do a POST request in a search query, returning a 400 response code. I

This is not correct, the search endpoint accepts both GET or POST request. Probably the reason it didn't work is that you are using Elasticsearch ≤ 6, and you also need to add `type: '_doc'` in the search parameters.

**Comment by mcapodici** [comment]
@delvedor - thanks for getting back to me.

I noticed there is quite a cool example here on https://www.elastic.co/blog/new-elasticsearch-javascript-client-released. I haven't tried it but it looks more comprehensive. Some of us are new to (or rusty on) both ES and the ES JS package, so something that does a lot of stuff and can be played with is always nice.

The ES version I was using was the 7.2.0 version for Windows, downloaded from https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.2.0-windows-x86_64.zip. I am admittedly using a different Java version and it gave me a warning. The version is `openjdk version "1.8.0_191-1-redhat"`

**Comment by delvedor** [comment]
Hello! Take a look at https://github.com/elastic/elasticsearch-js/pull/909, suggestions are welcome! :)

**Comment by delvedor** [comment]
Closed in #909.
